meetings/2023-10-18.md

```diff
@@ -63,23 +102,57 @@ URL: https://lists.w3.org/Archives/Public/public-solid/2023Oct/0036.html
 * SC: https://github.com/solid/authorization-panel/pull/331
 * SC: https://github.com/solid/webid-profile/pull/108
 * SC: https://github.com/solid/type-indexes/pull/28
-* SC: There are a few other documents that I didn't get to checking. Any objections?
+* SC: There are a few other documents that I didn't get to checking. Please review or Any objections or merge?
 * SC: This is also mentioned in https://github.com/solid/specification/blob/main/meetings/2023-10-11.md#wg-charter-proposal-status-update . See related https://github.com/solid/solid-wg-charter/pull/58 .
-
-
-### extend timeline
+* SC: I will leave them for a while. The CG does not use the Solid process any more. The focus of the Solid Process on the Director raised some issues with the AC review of the WG charter proposal.
+* eP: I'm just noticing that the WG charter proposal points to solidproject.org/TR/ . This needs to be updated as well.
+* SC: There is also an issue of using CG templates. WG charter could be updated to refer to EDs alternatively.
+* PAC: As I wrote in 58 "goes without saying" - the WG will follow the W3C Process and not other process. I'm not concerned about what those pointed documents say. They are just some input. The WG will create a new draft under the W3C Process based on that content. 
+* SC: Is it sufficient if we are updating the EDs rather than the ones in TRs?
+* PAC: There will be some text accompanying. In this text I can emphasis that the CG is using community-governance per CG charter. This move is occurring even if it is in the EDs, it is enough evidence for me.
+* SC: any objection to merging these PRs? All they do is remove the mention to the solid process?
+* eP: this is editorial change, so we can change only the third (patch) part of the version. Also, "editor's draft" is only for Working/Interest Groups. We should be using the CG templates.
+* SC: I've created an issue for that somewhere in w3c. They were supposed to update the process.
+* PAC: CG template mentions ED, e.g., https://www.w3.org/2021/12/rdf-star.html . Even if only mentioned for WGs, it wasn't the intention reserve that term.
```

Comment:
```suggestion
* SC: Is it sufficient if we update the EDs, rather than the ones in TR?
* PAC: There will be some accompanying text. In this text, I can emphasis that the CG is using community-governance, per CG charter. This move is occurring even if it is in the EDs; it is enough evidence for me.
* SC: Any objection to merging these PRs? All they do is remove the mention of the Solid process?
* eP: This is editorial change, so we can change only the third (patch) part of the version. Also, "editor's draft" is only for Working/Interest Groups. We should be using the CG templates.
* SC: I've created an issue for that somewhere in W3C. They were supposed to update the process.
* PAC: CG template mentions ED, e.g., [RDF-star and SPARQL-star: Final Community Group Report](https://www.w3.org/2021/12/rdf-star.html). Even if only mentioned for WGs, it wasn't the intention to reserve that term.
```